demos/sample_azure_iot_ca_recovery/sample_azure_iot_ca_recovery.c

```diff
@@ -135,6 +143,12 @@ uint64_t ullGetUnixTime( void );
 
 static uint8_t ucPropertyBuffer[ 32 ];
 static uint8_t ucScratchBuffer[ 128 ];
+static uint8_t ucRootCABuffer[ 5000 ];
```

Comment:
We should separate these in Recovery vs Operational.
Ideally we should create two structs within a single Union:

union {
operational_memory_struct,
recovery_memory_struct
}

This way the memory is still statically allocated but cannot be used in both modes at the same time and is shared between the two modes.